utensor_cgen/__main__.py

```diff
@@ -7,15 +7,26 @@
 
 from pip.commands.show import search_packages_info
 
+def _get_pb_model_name(path):
+  return os.path.basename(os.path.splitext(path)[0])
 
-def main(pb_file, src_fname, idx_dir, embed_data_dir, debug_cmt, output_nodes, version):
+def main(pb_file, src_fname, idx_dir, embed_data_dir, debug_cmt, output_nodes, version, model_dir):
```

Comment:
Because your review for my PR #4, the api of `main` has changed.
User should be able to choose different optimization method they want.

Plz rebase to latest `feature/ref_cnt` branch and solve the conflicts first.
I will test your code and discuss more with you later.

Reply:
@dboyliao done